cmd/main.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/opiproject/opi-spdk-bridge/pkg/frontend"
 	"github.com/opiproject/opi-spdk-bridge/pkg/kvm"
 	"github.com/opiproject/opi-spdk-bridge/pkg/middleend"
-	"github.com/opiproject/opi-spdk-bridge/pkg/server"
+	server "github.com/opiproject/opi-spdk-bridge/pkg/utils"
```

Comment:
would it be more readable to use `utils` as a package identifier instead of `server` if it even affects more lines?

Reply:
I don’t like to make huge diff in same PR

Reply:
We will use utils after propagation of this change to all bridges